source/tutorial/kotlin.txt

```diff
@@ -58,28 +58,62 @@ Prerequisites
 
 Start with the Template
 -----------------------
+
 This tutorial is based on the Kotlin SDK Flexible Sync Template App named 
 ``kotlin.todo.flex``. We start with the default app and build new features 
 on it. 
 
-From Realm CLI, you can run the :ref:`realm-cli apps create 
-<realm-cli-apps-create>` command to set up the backend and create the 
-Kotlin base app. The following command creates a new app based on the 
-kotlin.todo.flex template. With the options flags we can pass during creation,
-the command below creates an app:
+To learn more about the Template Apps, see :ref:`template applications <template-apps>`.
```

Comment:
Should we just use the title here? (Or otherwise titlecase this)
```suggestion
To learn more about the Template Apps, see :ref:`template-apps`.
```